### PR TITLE
add blazor page for adding a product

### LINKED ISSUE #45: As a product seller, I want to be able to add a product for sale

### PR DESCRIPTION
**Pull request by vinmaster**
closes #45

### CODE REVIEW

File: src/Congo.Blazor/Models/Product.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Congo.Blazor.Models
+{
+    public class Product
```
Comment (chrisK00) [inline src/Congo.Blazor/Models/Product.cs:8]: This is identical to the ProductResponse found in the .Contracts project

Comment (vinmaster) [reply src/Congo.Blazor/Models/Product.cs:8]: I tried changing it to `ProductResponse` in Contracts project and I get errors on `init` only properties. How would I get pass this?

Comment (chrisK00) [reply src/Congo.Blazor/Models/Product.cs:8]: If I am correct I believe @pobiega wanted to use different request models for the consumer of the API than the commands used by the handlers.
So could add the InsertProductRequest model to the Requests folder in the .Contracts project

**Review by cptcrunchy — APPROVED** [review]

**Comment by vinmaster** [comment]
I've updated it to use the contracts

File: src/Congo.Blazor/Pages/ProductAdd.razor [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿@page "/products/add"
+@using System.ComponentModel.DataAnnotations
+@using Congo.Contracts.Requests.Products
+
+<MudLink Href="/">Back to Home</MudLink>
+
+<EditForm Model=@Product OnValidSubmit=@Submit>
```
Comment (valincius) [inline src/Congo.Blazor/Pages/ProductAdd.razor]: Standard is to use quotes on all attributes

**Review by valincius — CHANGES_REQUESTED** [review]

File: src/Congo.Blazor/Pages/ProductAdd.razor (hunk 2 above)
Comment (vinmaster) [reply src/Congo.Blazor/Pages/ProductAdd.razor]: Is this okay now?

**Review by valincius — APPROVED** [review]

**Pull request merged**